phpBB/includes/ucp/ucp_register.php

```diff
@@ -561,8 +588,8 @@ function main($id, $mode)
 		$cp->generate_profile_fields('register', $user->get_iso_lang_id());
 
 		//
-		$this->tpl_name = 'ucp_register';
-		$this->page_title = 'UCP_REGISTRATION';
+		$this->tpl_name = $tpl_name;
+		$this->page_title = $page_title;
```

Comment:
It turns out that `$this->page_title` isn't used. The language var `UCP_REGISTRATION` doesn't even exist. The page title for registration is set to `REGISTER` in [ucp.php](https://github.com/phpbb/phpbb/blob/3.2.x/phpBB/ucp.php#L71) so trying to overwrite it in this event doesn't work. 

Could you remove `$this->page_title` all together please? (from this file and from the event)

Reply:
I think it is used here:
https://github.com/phpbb/phpbb/blob/master/phpBB/includes/functions_module.php#L968
but is not declared

Reply:
@rubencm I tested it and removing the field didn't change the title. (Changing it in ucp.php however worked). If it tries to fall back it also also can't work because the language var `UCP_REGISTRATION` doesn't exist.

Reply:
I see, you use 2 methods to set the titles, you use page_title, but it is overwritten with this:
https://github.com/phpbb/phpbb/blob/master/phpBB/ucp.php#L71
https://github.com/phpbb/phpbb/blob/master/phpBB/includes/functions_module.php#L1012
https://github.com/phpbb/phpbb/blob/master/phpBB/includes/functions_acp.php#L25

Reply:
Yes, line 71 of ucp.php is what I linked to in my first reply. 😛
Your third link is incorrect however, it's not going to the function `adm_page_header()` but to `page_header()`.

Reply:
Yes, you are right
I think all modules should use the $page_title method and remove the alternative methods